File: pset1/mario.c
#include <stdio.h>
#include <stdlib.h>
#include <ctype.h>
#include <cs50.h>

int main() {
    int height;                                             // The variable that contains the input number

    do {                                                    // Check the input number
        printf("height:\n");
        height = get_int();
    }
    while(height < 0 || height > 23 || isdigit(height));

    for(int i = 1; i <= height; i++){                        // The loop for number of rows

        for(int j = height - i; j > 0; j--)                  // The loop for number of spaces
            printf(" ");

        for(int k = i; k > 0; k--)                           // The loop for output "#"
            printf("#");

        printf("#\n");                                       // Output additional sign "#"
    }
}

File: pset6/mario.py
from __future__ import print_function
import cs50

height = 0

while height < 1 or height > 23:
    print("height: ")
    height = cs50.get_int()

for i in range(1, height+1):
    j = height - i
    k = i
    while j > 0:
        print(" ", end="")
        j = j-1

    while k > 0:
        print("#", end="")
        k = k-1

    print("#")
    